.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+test.md
```

Comment:
You have another `.gitignore` file inside the `.github/workflows` directory. Kindly remove it since you already have one in your root directory.